qiskit_ibm_provider/accounts/management.py

```diff
@@ -135,81 +132,67 @@ def get(
         cls.migrate()
         if name:
             saved_account = read_config(
-                filename=_DEFAULT_ACCOUNT_CONFIG_JSON_FILE, name=name
+                filename=cls._default_account_config_json_file, name=name
             )
             if not saved_account:
                 raise AccountNotFoundError(
                     f"Account with the name {name} does not exist on disk."
                 )
             return Account.from_saved_format(saved_account)
 
-        channel_ = channel or _DEFAULT_CHANNEL_TYPE
+        channel_ = channel or cls._default_channel_type
         env_account = cls._from_env_variables(channel_)
         if env_account is not None:
             return env_account
 
         if channel:
             saved_account = read_config(
-                filename=_DEFAULT_ACCOUNT_CONFIG_JSON_FILE,
-                name=cls._get_default_account_name(channel=channel),
+                filename=cls._default_account_config_json_file,
+                name=cls._default_account_name_ibm_quantum,
             )
             if saved_account is None:
                 raise AccountNotFoundError(f"No default {channel} account saved.")
             return Account.from_saved_format(saved_account)
 
-        all_config = read_config(filename=_DEFAULT_ACCOUNT_CONFIG_JSON_FILE)
-        for channel_type in _CHANNEL_TYPES:
-            account_name = cls._get_default_account_name(channel_type)
-            if account_name in all_config:
-                return Account.from_saved_format(all_config[account_name])
+        all_config = read_config(filename=cls._default_account_config_json_file)
+        account_name = cls._default_account_name_ibm_quantum
+        if account_name in all_config:
```

Comment:
`save()` has 
```
name: Optional[str] = _default_account_name,
``` 
So if I don't specify a `name`, `_default_account_name` (or "default") is used. But here it only looks for "default-ibm-quantum"

Reply:
you're right, it should default to None and then on line 48, it's set to "default-ibm-quantum"

```python
name = name or cls._default_account_name_ibm_quantum
```